vmm/src/cpu.rs

```diff
@@ -652,7 +652,7 @@ impl CpuManager {
         if let Some(snapshot) = snapshot {
             // AArch64 vCPUs should be initialized after created.
             #[cfg(target_arch = "aarch64")]
-            vcpu.lock().unwrap().init(&self.vm)?;
+            vcpu.init(&self.vm)?;
```

Comment:
I think you intended to include this change in the third commit for fixing the potential `vcpu` deadlock. Otherwise, this commit won't compile on aarch64.

Reply:
Oops, thank you!